### DISCUSSION TITLE
Failures in fork repo are notified to developers of upstream repo

### DISCUSSION THREAD

**Discussion by mame** [community/community#37095] [Actions]
I am experiencing that failures in GitHub Actions in a fork repository are sent to upsteam developers.

* The upstream repository that I am watching: https://github.com/ruby/ruby
* The fork repository whose failures are notified to me: https://github.com/turkdevops/ruby

This is not only for me, but for all watchers of ruby/ruby: https://github.com/turkdevops/ruby/issues/79. The notification is very annoying and I would like to stop it.

It appears that the fork repo is following the upstream changes by using a bot called https://github.com/apps/pull. The same issue is filed in the repository of the bot: https://github.com/wei/pull/issues/466

I am unsure why a notification in a fork repository is sent to upstream developers. Is this a GitHub issue, a bug of the bot, or an issue of our (upstream) configuration issue of GitHub Actions?

**Comment by ossanna16** [comment]
Hi there @mame and welcome to our community! Thank you for asking a great question 🙂
